server/controllers/customer.spec.js

```diff
@@ -0,0 +1,458 @@
+const sinon = require('sinon')
+const sandbox = sinon.createSandbox()
+import {omit} from 'lodash'
+const customerCtrl = require('./customer').default
+import Customer from '../models/customer'
+import User from '../models/user'
+import { clientRoles, questionnaireIdentifiers, customerStatus } from './../../common/constants'
+import Volunteer from '../models/volunteer'
+import Package from '../models/package'
+import {createUserSession, createTestUser} from '../tests/helpers'
+import Questionnaire from '../models/questionnaire'
+import mailer from '../lib/mail/mail-helpers'
+
+
+describe('Customer controller', function() {
+  describe('create', function() {
+    let req = {}
+    let res = {}
+    let user = null
+    let newCustomer = null
+    let userSession = null
+
+    before(async function() {
+      await initDb()
+      await Customer.find().remove()
+      await User.find().remove()     
+      await Questionnaire.find().remove()
+      await new Questionnaire({
+        name: 'Customer Application',
+        identifier: questionnaireIdentifiers.CUSTOMER
+      }).save()
+
+      newCustomer = createTestUser('user', clientRoles.CUSTOMER) 
+      userSession = await createUserSession(newCustomer) 
+      user = userSession.user
+    })
+
+    after(async function() {
+      await Questionnaire.find().remove()
+      await resetDb()
+      user = null
+    })
+
+    beforeEach(function() {
+      req = {
+        params: {},
+        body: {
```

Comment:
Most of `req.body`, `req.user` and `expectedResult` doesn't seem to be needed, I'd cut it down to only what the test is looking for.